patches.json

```diff
@@ -205,6 +205,39 @@
     "Fix pagebuilder module": {
       "2.3.1": "MDVA-22979__fix_pagebuilder_module__2.3.1.patch",
       "2.3.2": "MDVA-22979__fix_pagebuilder_module__2.3.2.patch"
+    },
+    "Fix XSS in order history": {
+      "2.2.0 - 2.2.8": "PRODSECBUG-2233__fix_xss_in_order_history__2.2.0.patch",
+      "2.3.0 - 2.3.1": "PRODSECBUG-2233__fix_xss_in_order_history__2.3.0.patch"
+    },
+    "Pass Store View scope in the Async/Bulk Web API": {
+      "2.3.1": "MAGETWO-99902__pass_store_view_scope_in async_web_api__2.3.1.patch",
+      "2.3.2": "MAGETWO-99902__pass_store_view_scope_in async_web_api__2.3.2.patch"
+    },
+    "Admin path disclosure bug": {
+      ">=2.1.4 <2.2.0": "PRODSECBUG-2432__admin_path_disclosure_bug__2.1.4.patch",
+      ">=2.2.0 <2.3.0": "PRODSECBUG-2432__admin_path_disclosure_bug__2.2.0.patch",
+      "2.3.0 - 2.3.2": "PRODSECBUG-2432__admin_path_disclosure_bug__2.3.0.patch"
+    },
+    "Pre-auth SQL Injection": {
```

Comment:
I think it's same as: https://github.com/magento/magento-cloud-patches/blob/develop/patches.json#L164

Reply:
yes, you are right. changes were in doc blocks only